test/functional/test_framework/util.py

```diff
@@ -339,7 +340,9 @@ def rpc_url(datadir, i, chain, rpchost):
             host, port = parts
         else:
             host = rpchost
-    return "http://%s:%s@%s:%d" % (rpc_u, rpc_p, host, int(port))
+    # TODO re-enable authentication
+    #return "http://%s:%s@%s:%d" % (rpc_u, rpc_p, host, int(port))
+    return "http://%s:%d" % (host, int(port))
```

Comment:
This is a topic we should have a discussion about. There has to be a username/password for accessing the node, probably. Also an authentication cookie, like bitcoin, would be a very good idea. Is authentication provided by `jsonrpsee`? I wonder.

Reply:
Added an issue about this #192